modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -38,29 +38,53 @@ function parseI3SNodeGeometry(arrayBuffer, tile = {}) {
   }
 
   const content = tile.content;
-  const {vertexAttributes, attributesOrder} = content.featureData;
+  const {
+    vertexAttributes,
+    attributesOrder,
+    featureAttributes,
+    featureAttributeOrder
+  } = content.featureData;
   // First 8 bytes reserved for header (vertexCount and featureCount)
-  const {vertexCount, byteOffset} = parseHeaders(content, arrayBuffer);
-  const {attributes} = normalizeAttributes(
+  const {vertexCount, byteOffset, featureCount} = parseHeaders(content, arrayBuffer);
+  const {attributes: normalizedVertexAttributes, byteOffset: offset} = normalizeAttributes(
```

Comment:
Maybe a few lines of comments here explaining what is going on. 

Maybe add a JSDoc header to this function?

Reply:
Added JDDoc + few lines of comments.